Commands/escaperooms.py

```diff
@@ -0,0 +1,548 @@
+import discord
+import pandas as pd
+import re
+import time
+import asyncio
+
+from datetime import datetime
+from discord.ext import commands
+from .GuildData import get_guild_data
+from Helpers.constants import *
+from Helpers.common_functions import usernames_from_ids
+from Translations.Translations import get_text as translate
+from Helpers.TranslationHelper import get_culture_from_context as culture
+from Helpers.emoji import white_check_mark, scales, trophy, thumbs_down, thumbs_up
+from Helpers.log import *
+
+
+class Escaperooms(commands.Cog, name="Escaperooms"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def read_tracking_table(self) -> pd.DataFrame:
+        """ Reads the Tracking Table from CSV and returns it as Pandas DF"""
+        convert_dict = {'Naam': str}
+        tracking_table = pd.read_csv(
+            'EscaperoomsCSV/tracking_table.csv', dtype=convert_dict)
+
+        tracking_table['Discord ID'] = tracking_table['Discord ID'].astype(
```

Comment:
First line reads the CSV into a Pandas dataframe
Second line makes sure the 'Discord ID' is in the dataframe in the correct formatting since pandas do not handle strings in CSVs that consist solely of numeric characters. If this line was not here it will always read it as a float no matter what the format in the actual CSV file.